### ISSUE TITLE
It does not build on a modern 64bit Fedora

### ISSUE THREAD

**Issue by doggodanubus** [Elzair/nazghul#1]
I'm not sure how to fix it.

**Comment by misutoneko** [comment]
Yeah looks like the compiler gets all confused with the min() and max() macros.
You need set the version first:

> export CFLAGS="-std=c++14"

**Comment by clort81** [comment]
https://github.com/gjones007/nazghul
I had success with this version.  Many changes i haven't reviewed.
Haxima has some particularly rough edges that are an itch to scratch.
